app/controllers/shopping_carts_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ShoppingCartsController < ApplicationController
+  before_action :get_shopping_cart
+
+  def show
+
+  end
+
+  private
+
+  def get_shopping_cart
+    @sum = 0
+    @cart = ShoppingCart.find(session[:cart_id])
```

Comment:
Wow, you guys really need to get into the habit of reading documentation.  Please revisit this.